### ISSUE TITLE
AioProcess kwargs has mutations

### ISSUE THREAD

**Issue by hellysmile** [dano/aioprocessing#14]
https://github.com/dano/aioprocessing/blob/master/aioprocessing/__init__.py#L29

```python

def AioProcess(group=None, target=None, name=None,
               args=(), kwargs = {}, *, daemon=None, context=None):
```

**Comment by hellysmile** [comment]
Just took a look, there are a lot similar issues

**Comment by dano** [comment]
Closed via #15
